### PR TITLE
update cli version and resolutions

### PR DESCRIPTION
**Pull request by JillieBeanSim**
Signed-off-by: Billie Simmons <49491949+JillieBeanSim@users.noreply.github.com>

## Proposed changes

update @zowe/cli version to 6.33.3 to fix the ssh2 audit failure.
add `"**/ansi-regex": "^5.0.1"` update to resolutions to fix audit failure
excluded "cpu-features" from webpack

image

## Release Notes

Milestone: 1.20.0

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [x] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Reminder

~After a PR is merged into the `master` branch, create a PR from `master` to the `next` branch resolving any conflicts.~ These updates are already being handled on next branch with the appropriate solutions.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/zowe-explorer/i18n/sample/src/Profiles.i18n.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "checkCurrentProfile.error": "Error encountered in ",
-  "addSession.quickPickOption": "Choose \"Create new...\" to define a new profile or select an existing profile to add to the DATA SETS Explorer",
+  "addSession.quickPickOption": "Choose \"Create new...\" to define or select a profile to add to the Job Views Explorer",
```
Comment (JillieBeanSim) [inline packages/zowe-explorer/i18n/sample/src/Profiles.i18n.json:3]: I have a question about this update that came in from `master` branch and showed up running `yarn workspace vscode-extension-for-zowe vscode:prepublish`

File: package.json [hunk 2]
```diff
@@ -18,8 +18,9 @@
     "vscode": "^1.43.0"
   },
   "dependencies": {
-    "@zowe/cli": "6.31.1",
-    "vscode-nls": "4.1.2"
+    "@zowe/cli": "6.33.3",
+    "vscode-nls": "4.1.2",
+    "node-loader": "^2.0.0"
```
Comment (lauren-li) [inline package.json:23]: With `cpu-features` excluded, are the `node-loader` pieces still needed here and in the webpack configs?

**Comment by lauren-li** [comment]
> @lauren-li in that case should the localization title, ie `addSession.quickPickOption`, have a diff. name for each tree? This way they don't keep overwriting each other?
> 
>     1. addSession.quickPickOption.uss
> 
>     2. addSession.quickPickOption.mvs
> 
>     3. addSession.quickPickOption.jobs

To be consistent with how we have named other tree-specific keys, I would suggest:

1. `uss.addSession.quickPickOption`
2. `ds.addSession.quickPickOption`
3. `jobs.addSession.quickPickOption`

**Comment by lauren-li** [comment]
> I always thought the labeling of the localization strings began with the method name, that's why I suggested the others @lauren-li

@JillieBeanSim You are correct about this. We do not have many cases where similar strings for different trees appear in the same function/file. The exception I had in mind when I made my suggestion was the Zowe Explorer package's `package.nls.json` file. (Given, that file doesn't use the the `ds` prefix, but it does use `uss` and `jobs` as prefixes, and `ds` is preferred over `mvs` when the Data Sets view is referenced in other Zowe Explorer IDs.)

**Review by lauren-li — APPROVED** [review]
This PR looks good to me! All unit and expected integration tests are passing, and I did not see any regressions in manual testing.

Thanks @JillieBeanSim for doing this update!

**Pull request merged**